app/src/main/res/drawable/circle.xml

```diff
@@ -0,0 +1,5 @@
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="oval">
+    <solid android:color="@color/md_red_500"/>
+    <size android:width="@dimen/color_picker_circle_size" android:height="@dimen/color_picker_circle_size"/>
```

Comment:
That pity line took me like 1 hour to debug, until I noticed, that it was missing :angry: